scheds/rust/scx_layered/src/stats.rs

```diff
@@ -576,13 +580,14 @@ impl SysStats {
 
         writeln!(
             w,
-            "busy={:5.1} util/hi/lo={:7.1}/{}/{} fallback_cpu/util={:3}/{:4.1}",
+            "busy={:5.1} util/hi/lo={:7.1}/{}/{} fallback_cpu/util={:3}/{:4.1} astall_preempts={}",
```

Comment:
I'd just say `antistall` and describe it as the number of tasks dispatched by the antistall mechanism.